experimental/dds/attributable-map/src/test/mocha/map.spec.ts

```diff
@@ -390,7 +390,6 @@ describe("Map", () => {
 				await map2.load(services2);
 
 				// Now connect the first SharedMap
-				dataStoreRuntime1.local = false;
```

Comment:
Should this be replaced with the new means of attaching?

Reply:
it was superfluous to begin with, just something copy pasted all over for no reason

Reply:
I see that the runtimes start out attached, but this seems contrary to what this (and many other tests in this PR) intend to test.  It looks like the logic to start them off in local state was removed in #4457 (maybe erroneously?):

https://github.com/microsoft/FluidFramework/commit/288b413f28562d71e7ee5456bc55f689ad24fa92#diff-b1677f9b488a7213ee2e6d128c3ff76750eb2c2470f159900628dfd25076dc66L237

Reply:
i can't say. maybe file a bug. we can bring back these lines. for now

Reply:
AB#7313, @agarwal-navin FYI